### PR TITLE
Allow multiple controllers and invokers in local environments.

### PR DESCRIPTION
**Pull request by markusthoemmes**
This allows deploying multiple controllers and invokers locally to be able to test their scaleout ability.

**Todo:**
- Add a variable `allow_multiple_instances` to be able to keep the double-invoker check for production environments.

**Depends on:**
- #2452 for getting rid of the registrator.
- #2507 to be able to deploy multiple invokers in one machine.

### CODE REVIEW

**Comment by rabbah** [comment]
nice!

**Comment by cbickel** [comment]
I found another dependency for this PR:
on generating the whisk.properties, two lists are generated. One with all controllers and one with all invokers.
On using these aliases, the list only contains the list of the aliases, but the IPs or hostnames are needed for the tests.

I found the following line in the internet: `controller.hosts={{ groups["controllers"] | map('extract', hostvars, 'ansible_host') | list | join(",") }}`
But on using this one, it does not work when using no aliases.
We should have a deeper look here.

**Comment by cbickel** [comment]
PG2#1973 is 🔵

File: ansible/logs.yml [hunk 1]
```diff
@@ -24,13 +24,14 @@
         - subjects
       when: "'db' not in exclude_logs_from"
 
-- hosts: shared,core,invokers
+- hosts: controllers,invokers
+  serial: 1
```
Comment (markusthoemmes) [inline ansible/logs.yml]: Is this change intentional?

Comment (cbickel) [inline ansible/logs.yml]: In the open source part there are neither `shared`-hosts nor `core`-hosts.
So I removed them, but added the controllers.

Comment (markusthoemmes) [inline ansible/logs.yml]: Any why the serial: 1?

Comment (cbickel) [inline ansible/logs.yml]: Wewere able to deploy several controllers on the same machine, because we changed the inventory_name of the machine.
Here it means, that this skript will be executed on all "hosts", which is only one physical host if we have only one machine.
I did not find a solution to execute this on all hosts, but only once if they have the same ansible_host.
So I did the workaround to execute it one after another to avoid race conditions when the machine copys the same log several times.
